.travis.yml

```diff
@@ -175,8 +76,3 @@ notifications:
     recipients:
       - ${ADMIN_EMAIL}
       - ${AUTHOR_EMAIL}
```

Comment:
I don't think the authors of commits should be notified of Travis status. I'm not sure about the usefulness of `COVERITY_SCAN_NOTIFICATION_EMAIL=${AUTHOR_EMAIL}` either. Would it be possible to just drop `AUTHOR_EMAIL` and `AUTHOR_NAME`?